_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,212 @@
+---
+id: 1e9941
+name: Keyboard shortcut is only active when component has focus
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters for a user interface component, then they are only available when that component has focus.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with [keyboard shortcuts][keyboard shortcuts].
+
+## Expectation
+
+For each [user interface component][] that is a [descendent][] of the root node of the test target, if [keyboard shortcuts][] are implemented using only [printable characters][], then they are only available when that component has focus.

```

Comment:
Is it possible for a shortcut to use multiple printable characters? Can `a+b` be a shortcut? (or are they going to be handled by separate events).

Reply:
The understanding document is not clear about this. [G217](https://www.w3.org/WAI/WCAG21/Techniques/general/G217) is also not clear on this. [F99](https://www.w3.org/WAI/WCAG21/Techniques/failures/F99) seems to apply to single key shortcuts only. 

@DagfinnRomen and @ShadowBB would you fail [SC 2.1.4](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html) if the keyboard shortcut required pressing `a+b`?

Reply:
> 
> 
> The understanding document is not clear about this. [G217](https://www.w3.org/WAI/WCAG21/Techniques/general/G217) is also not clear on this. [F99](https://www.w3.org/WAI/WCAG21/Techniques/failures/F99) seems to apply to single key shortcuts only.
> 
> @DagfinnRomen and @ShadowBB would you fail [SC 2.1.4](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html) if the keyboard shortcut required pressing `a+b`?

Yes I would fail that. And I think the SC itself is pretty clear on that as it mentions "characters" in plural.